### PR TITLE
Fix span timer test messaging and increase sleep

### PR DESCRIPTION
**Pull request by colin-higgins**
Changes proposed in this pull request:
Fix test message and up sleep time.

@DataDog/apm-dotnet

### CODE REVIEW

File: test/Datadog.Trace.Tests/SpanTests.cs [hunk 1]
```diff
@@ -128,11 +128,11 @@ public void Accurate_Duration(int minimumDurationMilliseconds)
 
             var averageElapsedTime = totalElapsedTime.TotalMilliseconds / iterations;
             var averageSpanTime = totalSpanTime.TotalMilliseconds / iterations;
-            var diff = Math.Abs(averageElapsedTime - minimumDurationMilliseconds);
+            var diff = Math.Abs(averageElapsedTime - minimumDurationMilliseconds); // Should be less than the threshold
 
             _output.WriteLine($"Average elapsed time: {averageElapsedTime:0.0} ms");
             _output.WriteLine($"Average span time: {averageSpanTime:0.0} ms");
-            Assert.True(diff < threshold, $"Span duration outside of allowed threshold. Expected: {minimumDurationMilliseconds}ms, actual average: {averageElapsedTime}ms");
+            Assert.True(diff < threshold, $"Span duration outside of allowed threshold. Expected max average of: {threshold}ms, actual average: {averageSpanTime}ms");
```
Comment (lucaspimentel) [inline test/Datadog.Trace.Tests/SpanTests.cs]:
```suggestion
            Assert.True(diff < threshold, $"Span duration outside of allowed threshold. Expected: {averageElapsedTime}ms, actual: {averageSpanTime}ms");
```

File: test/Datadog.Trace.Tests/SpanTests.cs [hunk 2]
```diff
@@ -128,11 +128,11 @@ public void Accurate_Duration(int minimumDurationMilliseconds)
 
             var averageElapsedTime = totalElapsedTime.TotalMilliseconds / iterations;
             var averageSpanTime = totalSpanTime.TotalMilliseconds / iterations;
-            var diff = Math.Abs(averageElapsedTime - minimumDurationMilliseconds);
+            var diff = Math.Abs(averageElapsedTime - minimumDurationMilliseconds); // Should be less than the threshold
```
Comment (lucaspimentel) [inline test/Datadog.Trace.Tests/SpanTests.cs]: This should be calculating the difference between the expected duration and the span duration.
```suggestion
            var diff = Math.Abs(averageElapsedTime - averageSpanTime); // Should be less than the threshold
```

File: test/Datadog.Trace.Tests/SpanTests.cs (hunk 1 above)
Comment (lucaspimentel) [inline test/Datadog.Trace.Tests/SpanTests.cs]:
```suggestion
            Assert.True(diff < threshold, $"Span duration outside of allowed threshold. Expected: {averageElapsedTime}ms, actual: {averageSpanTime}ms");
```

**Comment by colin-higgins** [comment]
Forced some failures to show the message: 
image

**Pull request merged**